mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -252,7 +301,7 @@ CLASS ail net/minecraft/entity/Entity
 	METHOD aO getMountedHeightOffset ()D
 	METHOD aP isLiving ()Z
 	METHOD aQ removeAllPassengers ()V
-	METHOD aR getBoundingBoxMarginForTargeting ()F
+	METHOD aR getTargetingDistance ()F
```

Comment:
Are you sure? If I remember correctly, this is the amount by which the bounding box is expanding, meaning:

  1. The new name could make people think it's from the center of the entity, as most distances are
  2. It's not an (euclidean) distance at all. It represents a cuboid region, not a spherical one as your name implies.

Reply:
I've abstracted this a bit. It's used in targetting to expand the bounding box, but effectively serves as an additional radius around there area an entity would normally be hit.

Maybe `getTargettingMargin` would be a better descriptor?

Reply:
Yes, margin is better. Though we decided on "targeting", not "targetting", in the last PR involving this a similar method since that's the spelling Mojang uses (see the F3 screen, "Targeted block").

Reply:
Yes, sorry. I always type targetting the way I've known it to be, even when I don't mean to. Same occurs with collissions.